tracing/README.md

```diff
@@ -0,0 +1,188 @@
+Distributed Tracing
+---
+
+Both Helidon SE and Helidon MP use Opentracing API for tracing
+events.
+
+_Eclipse Microprofile Opentracing specification is not yet implemented_
+
+#Usage
+## Usage In Helidon MP
+To use tracing in MP, simply depend on the `helidon-microprofile-tracing` module to automate
+setup of tracing, and add your favorite tracer implementation to the classpath (currently Zipkin is the only
+favorite tracer implemented, though SPI is available).
```

Comment:
"the only _favorite_ tracer implemented" --> I suggest dropping "favorite".  It gives the impression that other (non-favorite) tracer implementations are available :-)